serverHandlers/middleware.js

```diff
@@ -0,0 +1,48 @@
+'use strict';
+
+const UserModel = require('../Schemas/UserSchema');
+const fs = require('fs');
+let path = require('path');
+const allowedImgExts = [
+    "image/jpeg",
+    "image/jpg",
+    "image/gif",
+    "image/png"
+];
+
+exports.validateInputFields = async (req, res, next) => {
+    let userInfo = req.body;
+    let checkEmail = userInfo.email;
+    let imgPath = req.file.path;
+
+    UserModel.find({email: checkEmail}, (err, data) => {
+        if(data.length){
+            sendResponse("User already exists!", 409);
+        } else {
+            // validation check
+            if(!(/^([a-zA-Z]{1,32})$/.test(userInfo.name))){
```

Comment:
You have to store regexp in constants in a separate file.

Reply:
done